landlab/graph/framed_voronoi/framed_voronoi.py

```diff
@@ -67,73 +65,72 @@ def xy_of_node(
-            Minimum x-of-node and y-of-node values. Depending on the grid.
-            No node may be present at this coordinate.
+            Minimum *x*-of-node and *y*-of-node values. Depending on the grid,
+            a node may not be present at this location.
         xy_min_spacing: float or tuple of float, optional
             Final minimal spacing between nodes. Random moves of the core nodes
             around their initial positions cannot be above this threshold:
             ``(xy_spacing - xy_min_spacing) / 2``.  If ``float``, same minimal
-            spacing for x and y.
-        seed: tuple of int, optional
-            Seeds used to generate the random x and y moves.
-            When set, controls a pseudo-randomness of moves to ensure
-            reproducibility.
-            When None, seed is random and the moves of coordinates are
+            spacing for *x* and *y*.
+        seed: int, optional
+            Seed used to generate the random *x* and *y* moves. When set, controls a
+            pseudo-randomness of moves to ensure reproducibility.
+            When ``None``, seed is random and the moves of coordinates are
             completely random.
 
         Returns
         -------
         x_of_node, y_of_node : ndarray of float
-            The arrays of x and y coordinates.
+            The arrays of *x* and *y* coordinates.
 
         Examples
         --------
         >>> from landlab.graph.framed_voronoi.framed_voronoi import HorizontalRectVoronoiGraph
-        >>> HorizontalRectVoronoiGraph.xy_of_node((3, 3), seed=(200, 500))[0][3]       # doctest: +NORMALIZE_WHITESPACE
-        0.0
-        >>> HorizontalRectVoronoiGraph.xy_of_node((3, 3), seed=(200, 500))[0][5]       # doctest: +NORMALIZE_WHITESPACE
-        2.0
-        >>> HorizontalRectVoronoiGraph.xy_of_node((3, 3), seed=(200, 500))[1][3]       # doctest: +NORMALIZE_WHITESPACE
+
+        >>> x_of_node, y_of_node = HorizontalRectVoronoiGraph.xy_of_node((3, 3), seed=200)
+
+        Coordinates of the lower left node,
+
+        >>> x_of_node[0], y_of_node[0]
+        (0.0, 0.0)
+
+        *x* coordinates of the left and right edges,
+
+        >>> x_of_node[3], x_of_node[5]
+        (0.0, 2.0)
+
+        *y* coordinate of the middle row of the left edge,
+
+        >>> y_of_node[3]
         0.749
         """
 
         n_rows, n_cols = shape
-        max_move = [
+        max_move = (
             (xy_spacing[0] - xy_min_spacing[0]) / 2,
             (xy_spacing[1] - xy_min_spacing[1]) / 2,
-        ]
-        n_nodes = HorizontalRectVoronoiGraph.number_of_nodes(shape)
-        n_core_nodes = n_nodes - HorizontalRectVoronoiGraph.number_of_perimeter_nodes(
-            shape
         )
 
+        if max_move[0] < 0.0 or max_move[1] < 0.0:
+            raise ValueError("minimum spacing must be greater than node spacing")
+        if np.allclose(max_move, 0.0):
+            raise ValueError("at least one of x and y moves must be greater than zero")
+
```

Comment:
@sebastien-lenard The max moves can't be negative (i.e. the min spacing can't be greater than the initial spacing) but they can be zero in one, and only one, of the *x* or *y* directions. Does that see right to you?

Reply:
Yes, perfect.